etc/base.yaml

```diff
@@ -251,7 +247,7 @@ radar_appserver_postgresql:
           value:
     replication:
       applicationName: radarAppserver
-        # Uncomment when upgrading
+        # Uncomment when upgrading to v1.0.0
```

Comment:
These lines probably aren't needed anymore and can be removed

Reply:
I removed lines like this.